bears/setup.py.in

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+
+import locale
+
+import setuptools.command.build_py
+from setuptools import find_packages, setup
+
+try:
+    locale.getlocale()
+except (ValueError, UnicodeError):
+    locale.setlocale(locale.LC_ALL, 'en_US.UTF-8')
+
+long_description = '$LONG_DESCRIPTION'
+
+
+if __name__== "__main__":
+	setup(name='$NAME',
+		  version=$VERSION,
+		  description='The $NAME bear for coala (http://coala.rtfd.org/)',
+		  authors=$AUTHORS,
+		  authors_emails=$AUTHORS_EMAILS,
+		  maintainers=$MAINTAINERS,
+		  maintainers_emails=$MAINTAINERS_EMAILS,
+		  platforms=$PLATFORMS,
+		  license='$LICENSE',
+          long_description=long_description,
```

Comment:
what's wrong with adding the `$` variable right here? Also your indentation seems off